orchestration/dags/data_gcp_dbt/models/_dbt_docs/models/mart/global/description__mrt_global__special_event.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Special_event
+description: Description of the `mrt_global__special_event` table.
+---
+
+{% docs description__mrt_global__special_event %}
+
+# Special event Model
+
+The `mrt_global__special_event` table aims to register and analyse users' application to special events.
```

Comment:
expliquer ce qu'est un special event : 
 unique experiences developed for pass Culture beneficiaries in partnership with cultural actors, for which a lottery or a selection process based on motivation and objective criteria may be conducted.